src/gui/settingsdialog.h

```diff
@@ -89,6 +90,7 @@ private slots:
     // case the account avatar changes
     QHash<Account *, QAction *> _actionForAccount;
 
+    QMenuBar *_menuBar;
     QToolBar *_toolBar;
```

Comment:
@elsiehupp Is **QToolBar *_toolBar;** even needed anymore?

Reply:
I'm planning on testing reverting the switch from menubar to toolbar, since it may have messed things up. I'm not sure, though.

Reply:
It turns out this change was unnecessary. (It was also breaking the existing menu/toolbar.)